async-nats/CHANGELOG.md

```diff
@@ -1,3 +1,94 @@
+# 0.30.0
+## Overview
+This is a big release that introduces almost all breaking changes and API refinements before 1.0.0.
+The last two pending breaking items are: 
```

Comment:
My bad during editing the draft, it says last two, but I made it into three items.